packages/core/src/util/spanFilter.js

```diff
@@ -17,15 +17,18 @@ function shouldIgnore(span, endpoints) {
   if (!IGNORABLE_SPAN_TYPES.includes(span.n)) {
     return false;
   }
-  const operation = span.data?.[span.n]?.operation;
-
-  if (operation && endpoints[span.n]) {
-    const endpoint = endpoints[span.n];
-    if (Array.isArray(endpoint)) {
-      return endpoint.some(op => op === operation);
-    } else if (typeof endpoint === 'string') {
-      return endpoint === operation;
-    }
+  const endpoint = endpoints[span.n];
+  if (!endpoint) return false;
+
+  const operation = span.data?.[span.n]?.operation?.toLowerCase();
```

Comment:
suggestion: How about we add some comments?

This case looks like an internal usage bug, right?